guides/common/assembly_deployment-path.adoc

```diff
@@ -0,0 +1,40 @@
+include::modules/con_deployment-path.adoc[]
+
+include::modules/con_installing-a-server.adoc[leveloffset=+1]
+
+include::modules/con_project-server-with-external-database.adoc[leveloffset=+2]
+
+include::modules/con_configuring-dns-dhcp-and-tftp.adoc[leveloffset=+2]
+
+include::modules/ref_overview-of-authentication-methods-in-project.adoc[leveloffset=+1]
+
+include::modules/con_planning-organization-and-location-context.adoc[leveloffset=+1]
+
+include::modules/con_installing-smart-proxies.adoc[leveloffset=+1]
+
+include::modules/con_best-practices-for-deploying-smartproxies.adoc[leveloffset=+2]
+
+ifdef::satellite[]
```

Comment:
also affects Foreman and orcharhino **if** you run Foreman Server on RHEL. OK for now; I offer to create a follow-up PR that add a note for non-satellite with an explanation that it depends on the OS of your Foreman Server.

Reply:
Fixed with https://github.com/theforeman/foreman-documentation/pull/3520